src/pages/Detail/component/Creator.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+function Creator({ creatorName, profile_image }) {
+  const [moreTxt, setMoreTxt] = useState('더보기');
```

Comment:
```suggestion
  const [isOpenContent, setIsOpenContent] = useState(false);
```
state를 이처럼 변경 한 뒤 아래 있는 로직을 변화하는 방법이 좋겠습니다